ankerctl.py

```diff
@@ -32,10 +32,12 @@ class Environment:
     def __init__(self):
         pass
 
-    def require_config(self):
+    def load_config(self, required=True):
         with self.config.open() as config:
             if not getattr(config, 'printers', False):
-                log.critical("No printers found in config. Please import configuration using 'config import'")
+                log.warning("No printers found in config. Please upload configuration using the webserver or 'ankerctl.py config import'")
+                if required:
```

Comment:
We should use `log.critical` for this case. Something like:

```python
msg = "No printers found in config. Please upload configuration using the webserver or 'ankerctl.py config import'"
if required:
    log.critical(msg)
else:
    log.warning(msg)
```

Reply:
I can change that around. I was under the impression you didn't like the varying logic and wanted the message presented the same.

Reply:
✅